src/events.hpp

```diff
@@ -26,6 +26,7 @@
 #define DRAW_EVENT (SDL_USEREVENT + 3)
 #define CLOSE_WINDOW_EVENT (SDL_USEREVENT + 4)
 #define SHOW_HELPTIP_EVENT (SDL_USEREVENT + 5)
+#define TERMINATE_EVENT (SDL_USEREVENT + 6)
```

Comment:
This uses up the second-to-last definable SDL_USEREVENT (giving it the value of 30, max is 31).